### ISSUE TITLE
Reactors table in reactors howto is incorrect about what IOCP supports

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4629]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#4629|
|Type|defect|
|Created|2010-08-05 17:13:56Z|

IOCP supports UDP, processes and threads, but the table in the "Choosing Reactors" howto says it does not. The table should either be fixed, or possibly removed -- if everything says "Y" it's not clear it's good for much, and the notation about supported/experimental status could be moved into the descriptive paragraphs.

Attachments:

* [iocp-support-4629.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f15/f151e937fcb0b041f1b7a8519433173f2d9cfd94/687c0cd7f6b308601e1f54e39b0fd84b74c27a56.patch) (1720 bytes) - added by thijs on 2011-06-26 21:47:51Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4629 4629
type__defect defect
reporter__itamar itamar
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1281028436000000 1281028436000000
changetime__1309190165000000 1309190165000000
version__None None
owner__thijs thijs
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to PenguinOfDoom|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Patch up for review.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Thanks for working on this, thijs. It's helpful if you add the topfiles entry prior to review, but looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Replying to [jesstess](#note_5):
> Thanks for working on this, thijs. It's helpful if you add the topfiles entry prior to review, but looks good, please merge.

Good point, I tend to ignore them when it's a misc file but having someone make sure it's in the right `topfiles` folder during review is a good idea (r32191).

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [32193]) Apply iocp-support-4629.patch: Update the IOCP features in the Reactor Functionality table.

Author: thijs
Reviewer: jesstess
Fixes: [#4629](https://github.com/twisted/twisted/issues/4629)
